addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -17,22 +17,14 @@
 
 <!-- PAGE INDEX -->
 <!-- ============================================================ -->
-<template id="model_nav">
+<template id="forum_model_nav">
+    <!-- allows to be overridden such as in website_slides_forum -->
+    <t t-set="breadcrumb_kind" t-valuef="base"/>
     <nav id="o_wforum_nav" t-attf-class="navbar d-flex align-items-center gap-2 #{_page_name == 'single_question' and 'mw-xl-75 mw-xxl-100'} px-0" aria-label="breadcrumb">
         <t t-if="_page_name == 'single_question'">
             <div class="flex-grow-1">
-                <div class="row g-0">
-                    <ol t-if="_is_slide_channel" class="breadcrumb col-10 col-lg-6 mb-0 p-0">
```

Comment:
Should have been done directly in the first commit, but hey ... ok ^^ . Some remarks are therefore not applicable anymore.